natlas-server/webpack.config.js

```diff
@@ -1,9 +1,21 @@
 /* eslint-env node */
 const path = require('path');
+const webpack = require('webpack');
 const WebpackManifestPlugin = require('webpack-yam-plugin');
 const MiniCssExtractPlugin = require('mini-css-extract-plugin');
 const assetRootPath = path.resolve(__dirname, 'app', 'static');
 
+function revision() {
+    if (process.env.SOURCE_BRANCH === "main") {
+        return `main-${process.env.SOURCE_COMMIT}`;
+    }
+    const tag = process.env.DOCKER_TAG;
+    if (tag) {
+        return tag;
+    }
+    return "dev";
```

Comment:
Maybe we could signify this better than just `dev`? Can we get the `SOURCE_COMMIT` and then add `-dev` to the end of it or something?

Reply:
This case is limited to you running in your developer workspace. Is there any value in generating a useful version in that case? Ultimately that would require us to interface with Git.

Reply:
True, I suppose it's probably not that valuable.